uPortal-webapp/src/main/resources/properties/contexts/applicationContext.xml

```diff
@@ -91,6 +91,28 @@
     <!-- resolver that can be used to dynamically look up property values at run-time -->
     <bean id="propertyResolver" factory-bean="primaryPropertyPlaceholderConfigurer" factory-method="getPropertyResolver"/>
 
+    <!-- need to capture this value in a bean for map key, using el -->
+    <bean id="hibernateDialect" class="java.lang.String">
+        <constructor-arg value="${hibernate.dialect}"/>
+    </bean>
+
+    <!-- define default DDL used by most databases -->
+    <bean id="jgroupsDefaultInitSql" class="java.lang.String">
+        <constructor-arg value="CREATE TABLE JGROUPSPING (own_addr varchar(200) NOT NULL, cluster_name varchar(200) NOT NULL, ping_data blob DEFAULT NULL, PRIMARY KEY (own_addr, cluster_name) )"/>
```

Comment:
Just for readability, it might work (not sure) to do it like this....

```
<constructor-arg>
CREATE TABLE JGROUPSPING (
    own_addr varchar(200) NOT NULL, 
    cluster_name varchar(200) NOT NULL, 
    ping_data blob DEFAULT NULL, 
    PRIMARY KEY (own_addr, cluster_name))
</constructor-arg>
```

Reply:
Yeah, have to add <value></value> around it, apparently. Testing now...